Dockerfile

```diff
@@ -1,7 +1,8 @@
-# Build Stage: using Go 1.24.1 image
-FROM quay.io/projectquay/golang:1.24 AS builder
+# Build Stage: using Go 1.24.4 image
+FROM registry.access.redhat.com/ubi9/go-toolset:1.24.4 AS builder
```

Comment:
May we use just the 1.24 tag?
This way, we don't need to update it when the 1.25 is out.
wdyt?

Reply:
ok fair. I will update it